### ISSUE TITLE
Extension import follow up (import extension settings)

### ISSUE THREAD

**Issue by bsclifton** [brave/brave-browser#8847]
## Test plan
See https://github.com/brave/brave-core/pull/14997

## Description 
Follow up to https://github.com/brave/brave-browser/issues/7974

With https://github.com/brave/brave-core/pull/4672, ability to import extensions from Chrome is possible. The state of the extensions is not imported though.

## Steps to Reproduce

   1. Have an install of Chrome
   2. Install an extension that has state; ex: Metamask
   3. Use Brave's import functionality (added with https://github.com/brave/brave-core/pull/4672)

## Actual result:
State of extension is not imported

## Expected result:
State of extension should be imported

cc: @rebron @simonhong

**Comment by stephendonner** [comment]
## Verification `IN-PROGRESS` using 

Brave | 1.45.84 Chromium: 106.0.5249.91 (Official Build) beta (64-bit) 
--|--
Revision | fa96d5f07b1177d1bf5009f647a5b8c629762157-refs/branch-heads/5249@{#707}
OS | Linux

### Steps:
launched Google Chrome (`.`)
loaded `https://chrome.google.com/webstore/detail/metamask/nkbihfbeogaeaoehlefnkodbefgpgknn?hl=en`
clicked on `Add to Chrome`


Install Metamask to Google Chrome
Login/create wallet in MM
Import extensions from Chrome to Brave
Check that MM has same state in Brave as it was before in Chrome

**Comment by MadhaviSeelam** [comment]
Verification `PASSED` using
```
Brave | 1.45.84 Chromium: 106.0.5249.91 (Official Build) beta (64-bit)
-- | --
Revision | fa96d5f07b1177d1bf5009f647a5b8c629762157-refs/branch-heads/5249@{#707}
OS | Windows 11 Version 21H2 (Build 22000.978)
```
### Steps:

1. launched `Google Chrome` (`105.0.5195.127`)
2. visit `chrome.google.com/webstore` and search for `metamask`
3. click the `MetaMask `extension and click `Add to Chrome` button
6. clicked on `Add to Chrome`
7. clicked on the `Add extension` button
8. clicked on `Get started` within the `MetaMask` UI
9. clicked `No thanks`
10. clicked `Create a wallet`
11. pasted generated password
12. clicked `Create...`
13. shut down `Google Chrome`
14. launched `Brave`
15. clicked on `Let's go`
16. clicked on `Import from...`
17. chose `Chrome Person 1`
18. clicked `Import`
19. clicked `Done`
20. pasted the same password for MetaMask
21. clicked on `Unlock`

#### Confirmed successfully logged in to `Metamask` wallet imported from Chrome

wallet (Chrome) | re-auth screen | wallet (Brave)|Extension
--------------|------------|------|----------
<img width="488" alt="image" src="https://user-images.githubusercontent.com/98358127/193691591-b681dea0-3c66-46d8-8717-293bf304b5d8.png">|<img width="460" alt="image" src="https://user-images.githubusercontent.com/98358127/193691819-179db4e5-7076-4c9d-a2a3-73ce341b03d5.png">|<img width="461" alt="image" src="https://user-images.githubusercontent.com/98358127/193691854-0fa45395-13a5-4bc4-aeec-19ca55cf05d8.png">|<img width="462" alt="image" src="https://user-images.githubusercontent.com/98358127/193692492-a1ccdb05-94b2-45d5-b6ed-aec002f08ceb.png">

**Comment by ShivanKaul** [comment]
Hi folks, I can't get extension settings import to work on Brave. I tried following the steps on https://github.com/brave/brave-browser/issues/8847#issuecomment-1266107028 with source Chrome version 125.0.6422.142 and target Brave versions 1.67.115 Chromium: 126.0.6478.56  as well as 1.68.87 Chromium: 126.0.6478.56. I'm on macOS arm64.

In every test I did the extensions themselves were imported into Brave, but the settings did not carry over i.e. MetaMask did not give me the "Welcome back!" screen. I also tested uBO and did not see the custom filters get carried over. 
Is this a regression?
